src/applications/mhv/medical-records/util/txtHelpers/downloadRecords.js

```diff
@@ -137,7 +168,7 @@ ${record.notes.map(note => `${note}`)}
   const parseAllergies = records => {
     return `
 ${txtLine}
-Allergies
+4) Allergies
 
 If you have allergies that are missing from this list, send a secure message to your care team.
 ${records.map(
```

Comment:
All of these `records.map` calls are being joined with commas inside the string literals. You can put `.join('')` at the end to get rid of them (see Allergies.jsx) for an examples of this.
![image](https://github.com/department-of-veterans-affairs/vets-website/assets/87040148/145e703c-b094-4ecb-84ef-2a80a5ba50f1)

Reply:
Thanks for pointing out the oversight.